doc/pages/highlighters.asciidoc

```diff
@@ -102,7 +102,7 @@ add-highlighter window regex //\h*(TODO:)[^\n]* 0:cyan 1:yellow,red
     This highlights all the current search matches in italic:
 
 ---------------------------------------------
```

Comment:
can you fix this line length as well (add a `-` here and on the closing line line).